demisto_client/demisto_api/api/default_api.py

```diff
@@ -3797,26 +3796,22 @@ def get_widget_with_http_info(self, id, **kwargs):  # noqa: E501
             _request_timeout=params.get('_request_timeout'),
             collection_formats=collection_formats)
 
-    def import_classifier(self, file, **kwargs):  # noqa: E501
+    def import_classifier(self, file, classifier_id, **kwargs):  # noqa: E501
         """Import a classifier  # noqa: E501
 
         Import a classifier to Cortex XSOAR  # noqa: E501
         This method makes a synchronous HTTP request by default. To make an
         asynchronous HTTP request, please pass async_req=True
-        >>> thread = api.import_classifier(file, async_req=True)
+        >>> thread = api.import_classifier(file, classifier_id, async_req=True)
         >>> result = thread.get()
 
         :param async_req bool
         :param file file: file (required)
+        :param str classifier_id: associated typeID for the layout (required)
         :return: InstanceClassifier
                  If the method is called asynchronously,
                  returns the request thread.
         """
-        with open(file, 'r') as classifier_json_file:
```

Comment:
@glicht The classifier changes are a product of the codegen. Was this intentionally added, but not included in the `gen-code.sh` file?